common/src/main/java/com/wynntils/core/features/overlays/overlaySizes/FixedOverlaySize.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.core.features.overlays.overlaySizes;
+
+import com.wynntils.mc.utils.McUtils;
+
+public class FixedOverlaySize extends OverlaySize {
+    public FixedOverlaySize(float width, float height) {
+        super(width, height);
+    }
+
+    @Override
+    public float getWidth() {
+        return (float) (this.width / McUtils.window().getGuiScale());
+    }
+
+    @Override
+    public float getHeight() {
+        return (float) (this.height / McUtils.window().getGuiScale());
```

Comment:
I assume the gui scale thing works strangely, because on the surface of things it sounds like this would be the "gui scaled" overlay size, and the class below the "fixed" one...

Reply:
Yeah, it seems strange, but this works as intended. I got it mixed up first try too.

Reply:
Maybe add a comment acknowledging this f*ckedupedness of vanilla and asserting that this does indeed works as the name says..?